R/05_analysis_1.qmd

```diff
@@ -0,0 +1,82 @@
+
+```{r}
+    #| message: false
+    library(dplyr)
+    library("tidyverse")
+    rm(list = ls())
+
+
+diabetes_data <- read_tsv(file = "../data/04_diabetes_data_augment.tsv")
```

Comment:
Please add na = 'NA'

Reply:
It should also be 03_diabetes_data_augment.tsv and not 03_diabetes_data_augment.tsv

Reply:
Load the data in separate code chunk and add #| message:false